extensions/websockets-next/runtime/src/main/java/io/quarkus/websockets/next/InboundProcessingMode.java

```diff
@@ -12,12 +14,12 @@
 public enum InboundProcessingMode {
 
     /**
-     * Messages are processed serially, ordering is guaranteed.
+     * Events are processed serially, ordering is guaranteed.
```

Comment:
Should we also describe what an event is here?

Reply:
Or maybe just link to the Javadoc of the class

Reply:
Link to the javadoc of the class does not make sense IMO.

Reply:
kk